rakelib/github_stats.rake

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require 'httparty'
+require 'date'
+
+STATSD_ERROR_KEY = 'worker.github_stats_scraper.error'
+
+REPOS = %w[
+  vets-api
+  vets-website
+].freeze
+
+PR_KEYS = %w[
+  created_at
+  updated_at
+  head
+].freeze
+
+namespace :github_stats do
+  desc 'Hit the Github API and grab data on open PRs'
+  task get_open_prs: :environment do
+    # hit API endpoints for vets-api and vets-website to get open PRs and add data to array
+    responses = []
+    REPOS.each do |repo|
+      url = "https://api.github.com/repos/department-of-veterans-affairs/#{repo}/pulls?state=open&per_page=100"
+      resp = HTTParty.get(url)
```

Comment:
Probably can sub in `Faraday` for this so we keep down on deps

Reply:
Removed httparty and changed to:
resp = Net::HTTP.get(uri)
responses.concat(JSON.parse(resp))